### ISSUE TITLE
Docker image build broken from clean clone

### ISSUE THREAD

**Issue by jeffmaher** [cds-snc/node-starter-app#117]
# Description

Running a build of the Docker image from a clean checkout is broken.

# Recreation Instruction

This was done with:

- macOS 10.15.1
- nvm 0.35.0
- node 10.16.0
- npm 6.9.0

Instructions:

1. Clone this repo
1. Navigate into it
1. `docker build . -t cds-snc/node-starter-app`

Here's a copy/paste of my terminal:

```
jeff@providence (~/cds-snc)
>git clone git@github.com:cds-snc/node-starter-app.git
Cloning into 'node-starter-app'...
remote: Enumerating objects: 1911, done.
remote: Total 1911 (delta 0), reused 0 (delta 0), pack-reused 1911
Receiving objects: 100% (1911/1911), 634.72 KiB | 1.02 MiB/s, done.
Resolving deltas: 100% (1146/1146), done.

jeff@providence (~/cds-snc)
>cd node-starter-app

jeff@providence (~/cds-snc/node-starter-app)
>docker build . -t cds-snc/node-starter-app
Sending build context to Docker daemon  800.8kB
Step 1/8 : FROM node:10-alpine
 ---> 9dfa73010b19
Step 2/8 : LABEL maintainer="-"
 ---> Using cache
 ---> d5429cc20d35
Step 3/8 : ARG GITHUB_SHA_ARG
 ---> Using cache
 ---> 5962e1e6f058
Step 4/8 : COPY . /src
 ---> Using cache
 ---> f9a2662f90d8
Step 5/8 : WORKDIR /src
 ---> Using cache
 ---> f8f4eafb9a2c
Step 6/8 : RUN npm install --quiet --production
 ---> Running in ded9e95cdbdb
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/wrappy-ca4d4d6b/package.json'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/duplexify-946d51d5/package.json'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/cliui-bfae996b/package.json'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/wrappy-ca4d4d6b/README.md'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/duplexify-946d51d5/.travis.yml'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/cliui-bfae996b/CHANGELOG.md'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/duplexify-946d51d5/example.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/cliui-bfae996b/index.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/wrappy-ca4d4d6b/LICENSE'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/duplexify-946d51d5/index.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/wrappy-ca4d4d6b/wrappy.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/cliui-bfae996b/LICENSE.txt'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/duplexify-946d51d5/LICENSE'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/cliui-bfae996b/README.md'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/duplexify-946d51d5/README.md'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/source-map-35d0f70e/dist/source-map.min.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/duplexify-946d51d5/test.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/aws-sdk-5639b3e5/global.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/aws-sdk-5639b3e5/global.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/aws-sdk-5639b3e5/index.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/aws-sdk-5639b3e5/index.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/webpack-dev-server-8693e73f/lib/utils/processOptions.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/tar-e53ab210/.nyc_output/12bca2bc-6a2b-4f45-abc7-8483922a524d.json'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/aws-sdk-5639b3e5/LICENSE.txt'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/webpack-dev-server-8693e73f/client/utils/reloadApp.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/src/middlewares/check.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/webpack-dev-server-8693e73f/lib/utils/routes.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/nunjucks-e909b760/browser/nunjucks-slim.min.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/webpack-dev-server-8693e73f/lib/utils/runBonjour.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/src/chain/context-handler-impl.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/webpack-dev-server-8693e73f/lib/utils/runOpen.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/src/chain/context-handler.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/webpack-dev-server-8693e73f/client/utils/sendMessage.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/src/chain/context-runner-impl.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/src/chain/context-runner.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/check/index.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/webpack-dev-server-8693e73f/lib/utils/setupExitSignals.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/filter/index.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/webpack-dev-server-8693e73f/client/socket.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/src/chain/index.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/webpack-dev-server-8693e73f/client/sockjs.bundle.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/src/middlewares/one-of.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/stream-http-c2aba78e/test/server/static/test-polyfill.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/src/middlewares/sanitization-chain-builders.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/src/chain/sanitization-chain.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/src/middlewares/sanitize.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/src/chain/sanitizers-impl.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/src/chain/sanitizers.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/src/middlewares/schema.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/webpack-e9dfe829/lib/ChunkGroup.js'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/src/middlewares/validation-chain-builders.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/src/chain/validation-chain.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/express-validator-3cda3679/src/chain/validators-impl.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/src/node_modules/.staging/node-gyp-fed24f31/gyp/pylib/gyp/mac_tool.py'
npm ERR! path git
npm ERR! code ENOENT
npm ERR! errno ENOENT
npm ERR! syscall spawn git
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/cds-snc/webpack-starter.git
npm ERR! enoent 
npm ERR! enoent 
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-11-03T20_28_30_335Z-debug.log
The command '/bin/sh -c npm install --quiet --production' returned a non-zero code: 1

jeff@providence (~/cds-snc/node-starter-app)
>nvm --version
0.35.0

jeff@providence (~/cds-snc/node-starter-app)
>node --version
v10.16.0

jeff@providence (~/cds-snc/node-starter-app)
>npm --version
6.9.0
```

**Comment by timarney** [comment]
Looks like this is being caused by the webpack starter.

```
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/cds-snc/webpack-starter.git
```

I see about publishing that package to NPM tomorrow which I think will correct the issue.

Also thanks for the detailed notes 👍

**Comment by timarney** [comment]
@jeffmaher please give this another go with the latest update.

**Comment by timarney** [comment]
Released 5.0.4
